content/schema-resource/index.md

```diff
@@ -0,0 +1,118 @@
+---
+rfc: 0015
+start_date: 2018-02-09
+pr: openregister/registers-rfcs#28
+status: draft
+---
+
+# Schema resource
+
+## Summary
+
+This RFC proposes a schema resource to describe the register set of attributes
+with their datatypes.
+
+
+## Motivation
+
+The current mechanisms to get information about the schema are the `GET
+/register` endpoint or the Register register complemented with the Field
+register. It is cumbersome to get the full picture of the schema and, even
+more the day the schema gets the ability to evolve.
+
+
+## Explanation
+
+Note: All datatypes mentioned in this RFC are defined by the specification. In
+a few cases like `Cardinality` or `Datatype` it is described inline.
+
+***
+### Endpoint
+
+```
+GET /schema
+```
+
+### Parameters
+
+|Name|Type|Description|
+|-|-|-|
+|`log-size` | Optional `Integer`| Set the log size to the given entry number instead of the latest value.|
```

Comment:
Not sure I understand this param, is it to retrieve the schema at a particular point in the log?

If so, it would be great if the records endpoint could behave in the same way eventually.

Reply:
Yes, that's the purpose and I agree the data counterpart should have the same